_data/hosting.yml

```diff
@@ -173,6 +173,13 @@ websites:
       twitter: inmotionhosting
       img: inmotion.png
       tfa: No
+    
+     - name: iozoom
```

Comment:
Still 1 extra space 😉 
I'm not picky, it's just indentation is critical to yaml format and without everything being correct, the site won't generate the new changes.